postcss/theme.js

```diff
@@ -0,0 +1,180 @@
+const { Declaration, AtRule, Rule } = require('postcss');
+const supportedThemes = ['dark', 'light'];
+
+const getPropertyName = (selector, decl) => {
+    const regex = /([^A-Za-z0-9\-_])/g
+    return `--${selector}_${decl.prop}`
+        .replace(/\s+/g, '_')
+        .replace(regex, '\\$1')
+}
+
+const splitRule = (rule, selectorToExtract) => {
+    // |cloneAfter|, so in an |each| loop, the new rule will be processed.
+    const cloned = rule.cloneAfter();
+
+    // Remove the light selector from the cloned rule.
```

Comment:
This is actually removing the base selector, not necessarily `light`, right?

Reply:
Nice catch! I missed that when adding light theme support :laughing: